packages/credential-w3c/src/action-handler.ts

```diff
@@ -285,6 +288,20 @@ export class CredentialPlugin implements IAgentPlugin {
           },
         })
         verifiedCredential = verificationResult.verifiableCredential
+
+        // if credential was presented with other fields, make sure those fields match what's in the JWT
+        if (!(typeof credential === 'string')) {
```

Comment:
```suggestion
        if (typeof credential !== 'string') {
```

nitpicking, but it makes it easier to see the `!`